server/plugins/encryption/aes.go

```diff
@@ -0,0 +1,78 @@
+// Copyright 2022 Woodpecker Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package encryption
+
+import (
+	"crypto/cipher"
```

Comment:
please sort imports in all files, it should be in groups (separated by empty line):
* go bultin packages
* external packages
* project local packages

Reply:
**offtopic** @6543 @anbraten why do we have different order used that in other projects I have seen... usually it's:
* go bultin packages
* project local packages
* external packages

Reply:
ok, i will check all changed files. Unfortunately my IDE by default attempts to make it one big group, not 3 different

Reply:
done

Reply:
@lafriks good question, I guess because of default gofumpt behaviour?